### PR TITLE
Feature/checkbox disabled option

### PR DESCRIPTION
**Pull request by ixpd**
In this pr, was implemented the disabled option on the checkbox component

This feature will be used on the new component Checkbox tree

### CODE REVIEW

File: src/components/Inputs/Checkbox/Checkbox.vue [hunk 1]
```diff
@@ -58,17 +68,27 @@ export default {
...
+
         this.$emit('input', val);
       },
     },
   },
+
+  methods: {
+    getColor(color) {
+      const disabledColor = 'muted';
+      return this.disabled ? disabledColor : color;
```
Comment (Viserion77) [inline src/components/Inputs/Checkbox/Checkbox.vue:89]: Here we have a overConst

**Review by Viserion77 — APPROVED** [review]
LGTM

**Review by Viserion77 — APPROVED** [review]
LGTM kk

**Review by planivic1 — APPROVED** [review]
LGTM

**Pull request merged**